procrastinate/contrib/django/utils.py

```diff
@@ -0,0 +1,9 @@
+import pathlib
+
+
+def get_sql(name: str) -> str:
+    sql_path = migration_path = (
+        (pathlib.Path(__file__).parent.parent.parent) / "sql" / "migrations"
+    )
+    with open(migration_path / name, "rb") as f:
+        return f.read().decode()
```

Comment:
There's already a similar function in schema.py. Do you think:

- it would make sense to put this function there ?
- it would make sense to use `importlib_resources.read_text` ?

(My opinion is yes to both, but at this stage it's more of a discussion than a suggestion)